guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/main/java/org/guvnor/asset/management/client/editors/repository/wizard/CreateRepositoryWizard.java

```diff
@@ -64,36 +64,49 @@
 
     private final List<WizardPage> pages = new ArrayList<WizardPage>();
 
-    @Inject
     private RepositoryInfoPage infoPage;
 
-    @Inject
     private RepositoryStructurePage structurePage;
 
     private CreateRepositoryWizardModel model = new CreateRepositoryWizardModel();
 
-    @Inject
     private Caller<RepositoryService> repositoryService;
 
-    @Inject
     private Caller<RepositoryStructureService> repositoryStructureService;
 
-    @Inject
     private Caller<AssetManagementService> assetManagementService;
 
-    @Inject
     private Event<NotificationEvent> notification;
 
-    @Inject
     private KieWorkbenchACL kieACL;
 
-    @Inject
     private SessionInfo sessionInfo;
 
     private Callback<Void> onCloseCallback = null;
 
     private boolean assetsManagementIsGranted = false;
 
+    @Inject
+    public CreateRepositoryWizard( RepositoryInfoPage infoPage,
+            RepositoryStructurePage structurePage,
+            CreateRepositoryWizardModel model,
+            Caller<RepositoryService> repositoryService,
+            Caller<RepositoryStructureService> repositoryStructureService,
+            Caller<AssetManagementService> assetManagementService,
+            Event<NotificationEvent> notification,
+            KieWorkbenchACL kieACL,
+            SessionInfo sessionInfo ) {
```

Comment:
This is great! I very much like the constructor injection (instead of field injection)